torch/csrc/jit/passes/quantization.cpp

```diff
@@ -665,9 +665,44 @@ class InsertObserversHelper {
 
   // Is dynamic quantization enabled for the observer pass.
   bool is_dynamic = false;
+
   // These are the IR patterns we match to skip inserting observers.
   // They are compiled once on construction and used repeatedly within
   // the pass.
+
+  // nn.Linear + nn.ReLU
+  const PatternInfo nn_linear_nn_relu = PatternInfo::parse_from_str(R"(
```

Comment:
btw these patterns will need to be updated, please see the code in master to see how to do the update